tests/e2e/tests/devfiles/DevfileSmoke.spec.ts

```diff
@@ -22,6 +23,7 @@ const stack : string = 'Java Maven';
 suite(`${stack} test`, async () => {
     suite (`Create ${stack} workspace`, async () => {
         workspaceHandlingTests.createAndOpenWorkspace(stack);
+        CheReporter.registerRunningWorkspace(await WorkspaceNameHandler.getNameFromUrl());
```

Comment:
In this case, a workspace will be registered only when created&running. If the workspace will be created but fails to run, it will not be registered and will remain there after the test. Am I correct? At the same time, in the `fail` method, the registered workspace will be outdated (from the previously passed test) - will it fail as the test would try to stop&remove it?

Reply:
@Katka92 Are you sure about that?
```
    public createAndOpenWorkspace(stack: string) {
        test(`Open 'New Workspace' page`, async () => {
            await this.dashboard.waitPage();
            await this.dashboard.clickCreateWorkspaceButton();
            await this.createWorkspace.waitPage();
            await this.createWorkspace.clickOnSample(stack);
        });
    }
```
I believe it should be created and launched, but the tests will not wait for it to be running, they should get the workspace name as soon as it's created and STARTING